examples/dlp_api_example/main.tf

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2019 Google LLC
+ * Copyright 2018 Google LLC
```

Comment:
Why was this reverted to 2018? It should be 2019 throughout the repository.

Reply:
Nope, as checks will fail.

Reply:
2018 is fine.